packages/components/panel/panel.js

```diff
@@ -0,0 +1,10 @@
+export default () => {
```

Comment:
Relatively simple JavaScript but just wondering if it needs the same level of commenting as the rest.

eg. https://github.com/nhsuk/nhsuk-frontend/blob/master/packages/components/skip-link/skip-link.js

@tomdoughty may be help with the standardising of this.